plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/OpenNodeV2.java

```diff
@@ -0,0 +1,160 @@
+package io.everytrade.server.plugin.impl.everytrade.parser.exchange.bean;
+
+import com.univocity.parsers.annotations.Parsed;
+import com.univocity.parsers.common.DataValidationException;
+import io.everytrade.server.model.Currency;
+import io.everytrade.server.model.TransactionType;
+import io.everytrade.server.plugin.api.parser.FeeRebateImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.ImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.TransactionCluster;
+import io.everytrade.server.plugin.impl.everytrade.parser.ParserUtils;
+import io.everytrade.server.plugin.impl.everytrade.parser.exception.DataIgnoredException;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.ExchangeBean;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+import static io.everytrade.server.model.TransactionType.FEE;
+import static io.everytrade.server.model.TransactionType.WITHDRAWAL;
+import static io.everytrade.server.plugin.impl.everytrade.parser.ParserUtils.nullOrZero;
+import static java.util.Collections.emptyList;
+
+public class OpenNodeV2 extends ExchangeBean {
+
+    private String openNodeID;
+    private String typeOfTransfer;
+    private String statusOfTransfer;
+    private String date;
+    private String time;
+    private String amount;
+    private String transferFeesPaid;
+    private String currency;
+
+    @Parsed(field = "OpenNode ID")
```

Comment:
if you just setting value as string and not doing anything with this value, I think you can use this annotation directly on field and you dont have to have these setter methods. then it would look like this :

```
@Parsed(field = "OpenNode ID")
String openNodeID;
```

you can try this :)

Reply:
did you try it?